dff/messengers/telegram/abstract.py

```diff
@@ -0,0 +1,264 @@
+                    mime_type=update.animation.mime_type,
+                    thumbnail=thumbnail,
+                )
+            ]
+        if len(update.photo) > 0:
+            message.attachments += [
+                Image(
+                    id=picture.file_id,
+                    title=picture.file_unique_id,
+                    width=picture.width,
+                    height=picture.height,
+                ) for picture in update.photo
+            ]
+        if update.document is not None:
+            thumbnail = Image(id=update.document.thumbnail.file_id, title=update.document.thumbnail.file_unique_id) if update.document.thumbnail is not None else None
+            message.attachments += [
+                Document(
+                    id=update.document.file_id,
+                    title=update.document.file_unique_id,
+                    file_name=update.document.file_name,
+                    mime_type=update.document.mime_type,
+                    thumbnail=thumbnail,
+                )
+            ]
+
+        return message
+
+    def _create_keyboard(
+        self, buttons: Sequence[Sequence[Button]]
+    ) -> Optional[InlineKeyboardMarkup]:  # pragma: no cover
+        button_list = None
+        if len(buttons) > 0:
+            button_list = [
+                [
+                    InlineKeyboardButton(
+                        text=button.text, callback_data=button.data if button.data is not None else button.text
+                    )
+                    for button in row
+                ]
+                for row in buttons
+            ]
+        if button_list is None:
+            return None
+        else:
+            return InlineKeyboardMarkup(button_list)
+
+    async def cast_message_to_telegram_and_send(
+        self, bot: ExtBot, chat_id: int, message: Message
+    ) -> None:  # pragma: no cover
+        buttons = list()
+        if message.attachments is not None:
+            files = list()
+            for attachment in message.attachments:
+                if isinstance(attachment, Location):
+                    await bot.send_location(
+                        chat_id, attachment.latitude, attachment.longitude, reply_markup=self._create_keyboard(buttons)
+                    )
+                if isinstance(attachment, Contact):
+                    await bot.send_contact(
+                        chat_id,
+                        attachment.phone_number,
+                        attachment.first_name,
+                        attachment.last_name,
+                        reply_markup=self._create_keyboard(buttons),
+                    )
+                if isinstance(attachment, Poll):
+                    await bot.send_poll(
+                        chat_id,
+                        attachment.question,
+                        [option.text for option in attachment.options],
+                        reply_markup=self._create_keyboard(buttons),
+                    )
```

Comment:
Add support for sending attachments with extra fields + docs about those fields.